### PR TITLE
Single Runtime (remove devnet)

### PR DESCRIPTION
**Pull request by shawntabrizi**
This PR uses a single runtime crate with feature flags to build different flavors for Rococo, Kusama, and Polkadot.

As recommended by @bkchr

### CODE REVIEW

File: runtime/build.rs [hunk 1]
```diff
@@ -4,7 +4,23 @@ fn main() {
 		.with_current_project()
 		.export_heap_base()
 		.import_memory()
-		.build()
+		.build();
```
Comment (bkchr) [inline runtime/build.rs:7]: You don't need this. Or should this be the dev chain? Why not make the rococo the dev chain?

**Review by bkchr — APPROVED** [review]

File: runtime/build.rs (hunk 1 above)
Comment (shawntabrizi) [reply runtime/build.rs:7]: I thought this is the Polkadot chain? (the default build without feature flags)

Comment (shawntabrizi) [reply runtime/build.rs:9]: Added an explicit `polkadot-runtime` feature flag

**Comment by shawntabrizi** [comment]
Will work on `polkadot-runtime` explicit feature flag in another PR

**Pull request merged**